google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/StatementExecutor.java

```diff
@@ -151,18 +180,6 @@ private static ExecutorService createExecutorService() {
     this.interceptors = Collections.unmodifiableList(interceptors);
   }
 
-  /**
-   * Recreates this {@link StatementExecutor} and its {@link ExecutorService}. This can be necessary
-   * if a statement times out or is cancelled, and it cannot be guaranteed that the statement
-   * execution can be terminated. In order to prevent the single threaded {@link ExecutorService} to
-   * continue to block on the timed out/cancelled statement, a new {@link ExecutorService} is
-   * created.
-   */
-  void recreate() {
```

Comment:
This is no longer needed thanks to the new feature in the client library for setting a timeout per RPC.